packages/client/modules/teamDashboard/components/TaskColumn/TaskColumn.tsx

```diff
@@ -119,7 +122,18 @@ const TaskColumn = (props: Props) => {
             <StatusLabelBlock userCanAdd={userCanAdd}>
               <StatusLabel>{label}</StatusLabel>
               {tasks.length > 0 && <TasksCount>{tasks.length}</TasksCount>}
+              {status === DONE && (
+                <a onClick={togglePortal} className='ml-auto cursor-pointer text-sm text-slate-600'>
+                  Archive all
```

Comment:
-1 the UI needs a little update on mobile & tablet:

![Screenshot 2023-10-16 at 16 29 04](https://github.com/ParabolInc/parabol/assets/39854876/658cff23-1160-45f5-92cd-a2c2fe140215)


If there's not a good place for it, it could just be a desktop feature, and we'd add `hidden sm: block` to the className

Reply:
I added a `minWidth` to the `TaskColumn`:

<img width="513" alt="Screenshot 2023-10-25 at 10 21 27" src="https://github.com/ParabolInc/parabol/assets/1879975/adb183d0-ae31-483a-bb68-ac9f61c5a871">